Snappy/Response/PdfResponse.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Knp\Bundle\SnappyBundle\Snappy\Response;
+
+class PdfResponse extends SnappyResponse
+{
+    public function __construct($content, $fileName = 'output.pdf', $contentType = 'application/pdf', $contentDisposition = 'attachment', $status = 200, $headers = array())
```

Comment:
Well, standard and recommended is `application/pdf`, but there are also other [extravagant content types for PDF](http://stackoverflow.com/a/312258).